flow/designs/asap7/eth_top/config.mk

```diff
@@ -0,0 +1,50 @@
+$(info [INFO-FLOW] Ethernet Design)
+
+DESIGN_DIR                   := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+
+export DESIGN_NAME            = eth_top
+export DESIGN_NICKNAME        = eth_top
+export DESIGN                 = eth_top
+
+export VERILOG_FILES          = $(sort $(wildcard $(abspath $(DESIGN_DIR)/../../src/$(DESIGN))/*.v))
+export SDC_FILE               = $(DESIGN_DIR)/constraint.sdc
+export ABC_AREA               = 1
+
+export CORNER                ?= BC
+
+export LIB_FILES             += $($(CORNER)_LIB_FILES)
+export LIB_DIRS              += $($(CORNER)_LIB_DIRS)
+export DB_FILES              += $($(CORNER)_DB_FILES)
+export DB_DIRS               += $($(CORNER)_DB_DIRS)
+export WRAP_LIBS             += $(WRAP_$(CORNER)_LIBS)
+export WRAP_LEFS             += $(WRAP_$(CORNER)_LEFS)
+export TEMPERATURE            = $($(CORNER)_TEMPERATURE)
+export DFF_LIB_FILE = $($(CORNER)_DFF_LIB_FILE)
+
+export ABC_CLOCK_PERIOD_IN_PS = 400
+
+export DESIGN_POWER           = VDD
+export DESIGN_GROUND          = VSS
+
+export DESIGN_DIR
+
+
+export PLATFORM               = asap7
+export DESIGN_PDK_HOME
+
+export CORNER                ?= BC
+
+export PDN_CFG                = $(FOUNDRY_DIR)/openRoad/pdn/grid_strategy-M2-M5-M7.cfg
+
+export CORE_UTILIZATION       = 0.05
+export CORE_ASPECT_RATIO      = 1
+export CORE_MARGIN            = 2
+export PLACE_DENSITY          = 0.60
+
+export DONT_USE_SC_LIB         = $(OBJECTS_DIR)/lib/merged.lib
```

Comment:
why is this needed?

Reply:
Default CORE_UTILIZATION is 50. I don't have any hard macros, just allowing tool to use full core for placement. Otherwise congestion error will be there.

Reply:
@vijayank88 I think he's asking about `DONT_USE_SC_LIB`. @maliberty I believe it's a bad hack that's persisted because the ASAP7 standard cells are divided up into multiple `.lib`s so it needs to be merged into one `.lib` because of the outstanding issue with ABC being unable to use multiple `.lib`s.

There are multiple problems here.
1) This variable shouldn't be in the design config; it should be in the platform config. But that should get fixed in a different PR probably.
2) ABC should be able to handle multiple libs so we don't need to handle merging them together at all.
3) ASAP7 should honestly not be releasing fragmented libs like that. It is non-standard and makes no sense. I complained a while ago about it, but I don't think the complaint was forwarded to anyone able to address it.